test_util/helpers.py

```diff
@@ -31,6 +33,36 @@ def authenticate(self, cluster):
         logging.info('Authentication successful')
 
 
+def retry_boto_rate_limits(boto_fn, wait=2, timeout=60 * 60):
+    """Decorator to make boto functions resilient to AWS rate limiting and throttling.
+    If one of these errors is encounterd, the function will sleep for a geometrically
+    increasing amount of time
+    """
+    def ignore_rate_errors(*args, **kwargs):
+        local_wait = wait
```

Comment:
nit: this assumes (probably safe) that wait and timeout are value types / will always be copied rather than be reference things (strings, dictionaries, lists, etc). I like dropping a copy in this case just to make the target behavior very explicit and always work.